src/interfaces/node.h

```diff
@@ -173,6 +173,12 @@ class Node
     //! Get unspent outputs associated with a transaction.
     virtual bool getUnspentOutput(const COutPoint& output, Coin& coin) = 0;
 
+    //! Return default wallet directory.
+    virtual std::string getWalletDir() = 0;
+
+    //! Return available wallets in wallet directory.
+    virtual std::vector<std::string> listWalletDir() = 0;
```

Comment:
I find this function name confusing. How about `listWallets()`? A future version of this function could take a directory argument and the name would still make sense.

Reply:
I like the pairing between `getWalletDir` and `listWalletDir` but I don't have a strong preference. Let's see what others say.